content/en/docs/setup/install/multicluster/simplified/index.md

```diff
@@ -0,0 +1,142 @@
+
+These clusters shouldn't have Istio on them yet. Take a look at the [platform setup instructions](/docs/setup/platform-setup)
+for any special instructions for your particular environment.
+
+## Initial preparations
+
+You need to do a few one-time steps in order to be able to setup a multicluster mesh:
+
+1. Decide on what the name of your multicluster mesh will be. Something short but memorable is your best choice here:
+
+    {{< text bash >}}
+    $ export MESH_ID=mymeshname
+    {{< /text >}}
+
+1. Decide on the organization name to use in the intermediate certificates created to let the clusters communicate with one
+another. This should generally be derived from your organization's DNS name:
+
+    {{< text bash >}}
+    $ export ORG_NAME=mymeshname.mycompanyname.com
+    {{< /text >}}
+
+1. Create a working directory where to store a number of files produced during the cluster
+onboarding process:
+
+    {{< text bash >}}
+    $ export WORKDIR=mydir
+    $ mkdir -p ${WORKDIR}
+    $ cd ${WORKDIR}
+    {{< /text >}}
+
+1. Download the setup script to your working directory:
+
+    {{< text bash >}}
+    $ curl {{< github_file >}}/samples/multicluster/setup-mesh >./setup-mesh.sh
+    $ chmod +x ./setup-mesh.sh
+    {{< /text >}}
+
+1. Create a new [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts)
+and create contexts for each cluster:
+
+    1. Create a new KUBECONFIG file:
+
+        {{< text bash >}}
+        $ touch kubeconfig.yaml
+        {{< /text >}}
+
+    1. Set the `KUBECONFIG` environment variable to point to the configuration file you downloaded:
+
+        {{< text bash >}}
+        $ export KUBECONFIG=./kubeconfig.yaml
+        {{< /text >}}
+
+    1. Add the clusters to the configuration file:
+
+        {{< text bash >}}
+        $ TBD
+        {{< /text >}}
+
+    1. Create contexts for all three clusters:
+
+        {{< text bash >}}
+        $ TBD
+        {{< /text >}}
+
+1. And finally, prepare the mesh. This will create a root key and certificate that will
+be used to secure communication between the clusters in the mesh:
+
+    {{< text bash >}}
+    $ ./setup-mesh.sh create-mesh
+    {{< /text >}}
+
+{{< warning >}}
+The final step above created some secret material in your working directory, in particular some private keys associated
+with the certificate. You should store and protect those secrets. TBD: what files precisely?
+{{< /warning >}}
+
+## Customizing Istio
+
+Creating the mesh above created a file called `base.yaml` in your working directory. This file defines the
+basic configuration that will be used when deploying Istio in your clusters (which will happen below). You
+can customize the `base.yaml` file if you'd like to change how Istio will be deployed in the cluster.
+
+TBD: What can be customized? Pointer to existing docs?
```

Comment:
https://preliminary.istio.io/docs/setup/install/istioctl/#configure-the-feature-or-component-settings

The following parameters str overridden by the multi-cluster tool:

```
unvalidedValues.global.MyMesh0
values.gateway.istio-ingressgateway.env.ISTIO_MESH_NETWORK=<cluster's network name>
values.global.controlPlaneSecurityEnabled=true
values.global.multiCluster.clusterName=<Cluster name/UID>
values.global.network=<cluster's network name>
values.global.meshNetworks=<populated endpoints and gateways for all clusters in the mes>
values.pilot.meshNetworks=<populated endpoints and gateways for all clusters in the mes>
```